Modules/Core/Common/src/itkProcessObject.cxx

```diff
@@ -38,20 +38,10 @@ namespace itk
 
 namespace
 { // local namespace for managing globals
-constexpr size_t ITK_GLOBAL_INDEX_NAMES_NUMBER  = 100;
+constexpr size_t ITK_GLOBAL_INDEX_NAMES_NUMBER  = 10;
 constexpr size_t ITK_GLOBAL_INDEX_NAMES_LENGTH = 4;
```

Comment:
This should now be 3.

Reply:
yup... it's that way locally.

Reply:
@blowekamp can you push your new version with the size =3 so that we can push this first compromise through?